src/Microsoft.Health.Operations.Functions/Management/PurgeHistoryOptions.cs

```diff
@@ -43,4 +44,10 @@ public class PurgeHistoryOptions
     /// <value>A value cron expression</value>
     [Required]
     public string? Frequency { get; set; }
+
+    /// <summary>
+    ///  Gets or sets the collection of orchestration ids which should be skipped for cleanup
+    /// </summary>
+    /// <value>A set of one or more <see cref="string"/></value>
+    public IReadOnlyCollection<string>? InstancesToSkipPurging { get; set; } = Array.Empty<string>();
```

Comment:
nit: if this is nullable, then we probably don't need the default value to be empty. I think you can leave it as `null`

Reply:
Done